src/main/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Thanks for doing this Melanie!
My only question before merging, is if this is in the correct place. From a bit of research, it looks like the LICENSE or LICENSE.md file should live directly inside the project (as opposed to src/main, as it is here). That way, when you view the repository remotely, the license file shows up right away.

Reply:
fixed!